NitroxModel/Packets/WorldSending/AskWorldEvent.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NitroxModel.Packets.WorldSending
+{
+    class AskWorldEvent
```

Comment:
Can we remove the unused classes and add them back in when we start using the functionality?